### PR TITLE
Pandora fun

### PR DESCRIPTION
**Pull request by marcmengel**
So we needed a patch, but the way the build works we have to apply it in a weird way... 
Basically the package initially has almost no code in it and the ExternalPackage stuff
pulls it down, so we have to build part of it, apply the patch, and rebuild...

### CODE REVIEW

**Comment by marcmengel** [comment]
... and bxdecay0 updates.

File: packages/pandora/package.py [hunk 1]
```diff
@@ -65,12 +82,14 @@ def cmake_args(self):
             ),
             "-DCMAKE_MODULE_PATH={0}/etc/cmake".format(self.spec["root"].prefix),
             "-DPANDORA_MONITORING=ON",
-            "-DPANDORA_EXAMPLE_CONTENT=OFF",
-            "-DPANDORA_LC_CONTENT=OFF",
-            "-DPANDORA_LAR_CONTENT=OFF",
+            "-DLC_PANDORA_CONTENT=ON",
+            "-DLAR_PANDORA_CONTENT=ON",
+            "-DiINSTALL_DOC=OFF",
```
Comment (greenc-FNAL) [inline packages/pandora/package.py:87]: Should that be
```.py
            "-DINSTALL_DOC=OFF",
```
?

**Review by greenc-FNAL — CHANGES_REQUESTED** [review]
I would suggest we document the reason for the patch and any applicable bug reports or other links too.

File: packages/pandora/package.py (hunk 1 above)
Comment (marcmengel) [reply packages/pandora/package.py]: Updated as requested.

**Pull request merged**